internal/storage/storage_handle_test.go

```diff
@@ -51,7 +51,8 @@ func (testSuite *StorageHandleTest) TearDownTest() {
 
 func (testSuite *StorageHandleTest) TestBucketHandleWhenBucketExistsWithEmptyBillingProject() {
 	storageHandle := testSuite.fakeStorage.CreateStorageHandle()
-	bucketHandle := storageHandle.BucketHandle(TestBucketName, "")
+	ctx := context.Background()
```

Comment:
nit: Code simplification.

Add this to SetupTest() 

```go
testSuite.ctx = context.Background()
```

And then just use `testSuite.ctx` directly in each test function.

Reply:
Updated.